### PR TITLE
ddl: Support to modify column with default value and comment

### PR DESCRIPTION
**Pull request by zimulala**
Handle #2598

### CODE REVIEW

File: ddl/ddl_db_test.go [hunk 1]
```diff
@@ -733,6 +733,14 @@ func (s *testDBSuite) TestChangeColumn(c *C) {
 	s.tk.MustQuery("select a from t3").Check(testkit.Rows("1", "2"))
 	s.mustExec(c, "alter table t3 change a aa bigint")
 	s.tk.MustQuery("select aa from t3").Check(testkit.Rows("1", "2"))
+	s.mustExec(c, "alter table t3 modify b varchar(20) default 'c' comment 'my comment'")
```
Comment (coocood) [inline ddl/ddl_db_test.go]: Use InfoSchema to get the table, verify that the comment is updated.

**Comment by coocood** [comment]
LGTM

**Comment by zimulala** [comment]
PTAL @XuHuaiyu @hanfei1991

**Comment by XuHuaiyu** [comment]
LGTM

**Pull request merged**